PostgreSQL/connector.json

```diff
@@ -0,0 +1,44 @@
+{
+	"name": "PostgreSQL",
+	"fileName": "connector.js",
+	"fields": [
+		{
+			"type": "String",
+			"key": "host",
+			"label": "Host",
+			"htmlInputType": "text",
+			"encrypted": false,
+			"required": true
+		},
+		{
+			"type": "String",
+			"key": "port",
+			"label": "Port",
+			"htmlInputType": "text",
+			"encrypted": false,
+			"required": true
+		},
+		{
+			"type": "String",
+			"key": "user",
+			"label": "User",
+			"htmlInputType": "text",
+			"encrypted": false,
+			"required": true
+		},
+		{
+			"type": "String",
```

Comment:
Type here might not be `String`. Can you check on the marketplace UI?

Reply:
Passwords are also string. And encrypted.